digital-credentials/dc-types.ts

```diff
@@ -33,9 +33,20 @@ export interface MobileDocumentRequest {
  */
 export interface MakeGetOptionsConfig {
   /**
-   * Protocol(s) to use for the request
+   * Protocol(s) to use for the request.
+   * Can be a single protocol, array of protocols, or empty array.
+   * If not provided, uses the default supported protocol.
    */
   protocol?: GetProtocol | GetProtocol[];
+  /**
+   * Explicit credential requests.
+   * When provided, these are used in addition to any protocol-based requests.
+   */
+  requests?: DigitalCredentialGetRequest[];
+  /**
+   * Optional data to override canonical data for protocol-based requests.
+   */
+  data?: MobileDocumentRequest | object;
```

Comment:
I recall that some tests are testing passing here invalid data, which aren't necessarily objects.
Maybe we should make this to capture that?

```suggestion
  data?: MobileDocumentRequest | any;
```